src/App.js

```diff
@@ -1,29 +1,9 @@
-import logo from './logo.svg';
+/* eslint-disable */
 import './App.css';
+import calculator from './components/calculator';
 
 function App() {
```

Comment:
- [ ] Like the previous CR requested please make your `App` component class-based, right now it is still a functional component.

Reply:
This change has been done well